src/modules/shortcut_guide/shortcut_guide.cpp

```diff
@@ -52,6 +54,16 @@ namespace
         {
             return result;
         }
+
+        DWORD pid;
+        WCHAR exePath[MAX_PATH]; 
+        GetWindowThreadProcessId(active_window, &pid);
+        HANDLE processHandle = OpenProcess(PROCESS_QUERY_INFORMATION, FALSE, pid);
+        GetProcessImageFileName(processHandle, exePath, MAX_PATH);
+        CloseHandle(processHandle);
+
+        result.disabled = instance->is_disabled_app(exePath);

```

Comment:
if `GetProcessImageFileNameA` failed, we're passing garbage data here. we need to initialize `exePath` like this:

```
WCHAR exePath[MAX_PATH]{};
```
and only call `is_disabled_app` if `GetProcessImageFileName` didn't fail, as well as other functions like `OpenProcess`